service/controller/clusterapi/v19/cluster_resource_set.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 
 	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	appresource "github.com/giantswarm/apiextensions/pkg/resource/app"
```

Comment:
Naming is a bit weird here. The resources in the operatorkit repo have the resource suffix. Should we change this?

Reply:
Why is this resource even in `apiextensions`? There should only be types and clients in there. Nothing operatorkit related. I get why we do not put it in `operatorkit` but why is this not implemented where it is used? I would vote for a separate project when different operators need the resource, or simple copy/paste as with all the namespace resources and whatnot.

Reply:
We need the resource in app-operator and release-operator. In future more operators might need to create app CRs. But yet I'm not keen on using apiextensions for this and would prefer a separate repo.

For the package structure what about this? `github.com/giantswarm/operatorkitresources/pkg/appresource`

Reply:
Let's not put all the resources into the same repo again. We have so many problems already with mixed dependencies. I would say the resource could just have its own repo for now. When there is more to pack or add we can extend it once we have an idea.

Reply:
Ack. Yes, I think `github.com/giantswarm/appresource` is better.

Reply:
I don't understand what are problems with having that in apiextensions and having all resources in a single repo. We already agreed on this approach during SIG operator sync. But I can name problems with having too many repos. Can you give some examples of the problems here?